feat/src/main/java/com/example/feat/document/controller/DocumentViewModel.kt

```diff
@@ -91,8 +94,9 @@ class DocumentViewModel @Inject constructor(
 
     private fun updateStateWithTutors(either: Either<ExceptionState, Any>) {
         if (either.isRight()) {
-            val tutors = (either.rightValue() as TutorFavorites).tutors
-            val favoriteTutors = (either.rightValue() as TutorFavorites).favoriteTutors
+            val tutorResponse = (either.rightValue() as TutorFavorites)
+            val tutors = tutorResponse.tutors
+            val favoriteTutors = tutorResponse.favoriteTutors
```

Comment:
Casting the result of 'either.rightValue()' multiple times for the same type is inefficient. Consider storing the casted object in a variable to improve performance.

```diff
- val tutorResponse = (either.rightValue() as TutorFavorites)
+ val tutorResponse = either.rightValue() as TutorFavorites
```